src/openvpn/route.c

```diff
@@ -300,6 +332,19 @@ is_special_addr(const char *addr_str)
     }
 }
 
+bool
+ipv6_is_special_addr(const char *addr_str)
+{
+    if (addr_str)
```

Comment:
any reason not to move this if into the function instead?

Reply:
I agree. This way we can get rid of one level of invocations. I know the ipv4 code is probably doing something similar, but I'd vote for simplifying the new code (and possibly the old one too later on), rather than introducing more "bad" code.